### ISSUE TITLE
Automatic Payments Cancel/Re-Request When Price Changes

### ISSUE THREAD

**Issue by chris206** [206Tours/DB#2052]
IF automatic/scheduled payments cannot be modified when clients increase or decrease their pilgrimage costs - then we need to cancel, re-run numbers, and re-request in the event prices change (if the change has  been registered for at least 24 hours.)

Otherwise, the majority of orders will under/overcharge and we will never get a handle on it.

**Comment by chris206** [comment]
I think my idea has a limitation - the creation of a new request would mean that the card would be stored on our end, right? This could lead to liability if we don't do it securely and would be more complicated than just figuring out how to automatically update the charge amount on an existing record.

**Comment by bradatkins** [comment]
No...it would not mean that. We would just send a request to authnet and
say create a monthly subscription of 1 month, starting in Dec 2017 for
example

**Comment by chris206** [comment]
I think you're replying to the wrong github - this one is about cancelling and re-requesting when there is a discrepancy. The other GH is about #2055 

My fault for creating 2 ongoing discussions about similar things
